### PR TITLE
updated Pioneer 20 and added Papa and Irm 11

### PR DESCRIPTION
**Pull request by rtravis-whoi**
- Updated Pioneer 20 start/end dates/times - start cruise elog entry attached; SSSGs ended elog early at end of Leg 2 and put an eta arrival time, but that isn't accurate for actual end of cruise.  At dock at 10:02 local (14:02 UTC) on 21 April. 

- Added preliminary Papa 11 and Irminger 11 start/end dates/times:
- CUIDs and start/end dates below confirmed by Dee...
           Papa 11 – SKQ202408S; 28 May – 15 June
           Irminger 11 – AR84-01; 31 May – 2 July
AR82start

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Pioneer 20 AM Updates](https://trello.com/c/22LKGqi6/232-pioneer-20-am-updates)

**Comment by rtravis-whoi** [comment]
[Irminger 11 AM Updates](https://trello.com/c/jAVA6WUL/237-irminger-11-am-updates)

**Review by cooleyky — APPROVED** [review]
looks good!

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -340,4 +340,6 @@ EK20231218,Elakha,2023-12-18T15:30:00,2023-12-18T21:09:00,Endurance glider recov
 SU20240213,Pacific Surveyor,2024-02-13T17:46:00,2024-02-14T00:31:00,Endurance glider deployments 320-14 & 381-13
 EK20240308,Elakha,2024-03-08T17:20:00,2024-03-08T22:00:00,Endurance ce02shsp-00034 CSPP deployment
 SKQ202405S,Sikuliaq,2024-03-29T23:00:00,2024-04-12T00:00:00,Endurance 20
-AR82,Neil Armstrong,2024-03-29T00:00:00,2024-04-21T23:59:00,Pioneer 20 (MAB)
+AR82,Neil Armstrong,2024-04-01T13:20:00,2024-04-21T14:02:00,Pioneer 20 (MAB)
+SKQ202408S,Sikuliaq,2024-05-28T00:00:00,2024-06-15T23:59:00,Papa 11
+AR84-01,Neil Armstrong,2024-05-31T00:00:00,2024-07-02T23:59:00,Irminger 11
```
[the remark this answers is not in the document]
Comment (snwhite) [reply cruise/CruiseInformation.csv:345]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**